src/dataset-builder/CohortEditor.ts

```diff
@@ -475,7 +475,7 @@ const CohortGroups: React.FC<CohortGroupsProps> = (props) => {
       : div([
           _.map(
             ([index, criteriaGroup]) =>
-              h(Fragment, { key: criteriaGroup.name }, [
+              h(Fragment, { key: index + 1 }, [
```

Comment:
There's three uses of `index + 1` here. If they're all in the same scope, you could create a variable for it to avoid repeating the expression, something like `userIndex`.